db/migrate/20190220091825_add_biography_to_author.rb

```diff
@@ -0,0 +1,6 @@
+class AddBiographyToAuthor < ActiveRecord::Migration[5.2]
+  def change
+  	add_column :authors, :biography, :text
+  	add_column :authors, :github_id, :integer
```

Comment:
Looks like you have some weird formatting (whitespace) in this file.

Reply:
Removed space